package.json

```diff
@@ -144,6 +144,7 @@
     "rollup-plugin-node-globals": "^1.4.0",
     "rollup-plugin-node-polyfills": "^0.2.1",
     "rollup-plugin-visualizer": "^4.1.0",
+    "sass": "^1.39.2",
```

Comment:
[Structure] `sass` is a pretty big dependency to use for exactly one `@extend`. I don't think it's justified here. Let's revert the change from `.css` to `.sass`, unless there's a strong reason for it?

Reply:
Removed in #5218 ;)